javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,81 @@
+let displayName = "";
+let toDoList = [];
+
+function getReply(command) {
+    const arrayName = command.split(" ");
+    if (command.includes("Hello my name is")) {
+        displayName = arrayName[4];
+        return `Nice to see you ${displayName}`;
+    } else if (command.includes("What is your name")) {
+        return `My name is ${displayName}`;
+    }
+    if (command == "") {
+        return "Please say your name";
+    }
+    if (command.includes(arrayName)) {
+        return `Hey ${displayName} i know you`;
+    } // adding and removig to todoList
+    else if (command == "Add fishing to my todolist") {
+        toDoList.push("fishing");
+        return "fishing added to todolist";
+    } else if (command == "Add singing in the shower to my todolist") {
+        toDoList.push("singing in the shower");
+        return "singing in the shower added todolist";
+    } else if (command == "Remove fishing from my todolist") {
+        toDoList.shift();
+        return "Removed fishing from your todolist";
+    } else if (command == "What is on my todolist?") {
+        toDoList.unshift("fishing");
+        return toDoList;
+    }
+    // To display what day is today.
+    if (command == "What day is it today?") {
+        const days = [
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+            "Sunday",
+        ];
+
+        const today = new Date();
+        const daypattern = { day: "numeric", month: "long", year: "numeric" };
+
+        return today.toLocaleDateString(undefined, daypattern);
+    } else {
+        return "I dont understand what you are saying";
+    }
+}
+//function for simple math.
+function airthOperator(command) {
+    let resultNum = command.match(/\d/g);
```

Comment:
Note
```js
"15 + 23".match(/\d/g) // => (4) ["1", "5", "2", "3"]
"15 + 23".match(/\d+/g) // => (4) ["15", "23"]
```
`+` means one or more digits (`\d`)

But you can split the command string into an array. It could be easier to you
```js
      const words = command.split(' ');
      const x = parseInt(words[2]);
      const y = parseInt(words[4]);
```